tests/c/simplecall.newcg.c

```diff
@@ -0,0 +1,67 @@
+// ignore-if: test $YK_JIT_COMPILER != "yk" -o "$YKB_TRACER" = "swt"
+// Run-time:
+//   env-var: YKD_PRINT_IR=aot,jit-pre-opt
+//   env-var: YKD_SERIALISE_COMPILATION=1
+//   env-var: YKD_PRINT_JITSTATE=1
+//   status: error
+//   stderr:
+//     jit-state: start-tracing
+//     foo
+//     jit-state: stop-tracing
+//     --- Begin aot ---
+//     ...
+//     func main($arg0: i32, $arg1: ptr) -> i32 {
+//     ...
+//     --- End aot ---
+//     --- Begin jit-pre-opt ---
+//     ...
+//     {{1}}: i8 = Icmp {{2}}, SignedGreater, 1i32
```

Comment:
Can we use `%{{}}` for local variable placeholders? This prevents constant operands from matching where we expect a variable operand.

Reply:
[262fb21](https://github.com/ykjit/yk/pull/1082/commits/262fb214025ac8ac85b65d4ecc542e8630345937)